metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Transitive release date control for child objects of a Study
+
+  Scenario Outline: Objects to which at least released study links should be accessible, and vice versa
+
+    # Create the common taxonomy
+    When I request POST taxonomies with 9606 for ID, Homo Sapiens for name and NONE for ancestors
+    Then set the URL to TAXONOMY
+    And the response code should be 201
+
+    # Create the common reference sequences
```

Comment:
This block should be like:

1. When I request POST /reference-sequences with JSON-like payload (...)
2. And set the URL (...)
3. Then the response code should be 201

This template can be used as reference for others below.

Reply:
Thank you for the clarification, looking at existing tests I couldn't quite determine what is the accepted practice. Now rewrote, fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/2da2671086076bae4c89bb2b6f79b4363089ef75.